charts/internal/calico/values.yaml

```diff
@@ -39,5 +39,7 @@ images:
 
 vpa:
   enabled: false
+  node: false
+  typha: false
```

Comment:
I'm wondering, why there are two new fields here, that are controlled by just one flag in the network configuration.
Is there a reason for that?

Reply:
For granularity, to decide on vpa or cluster-proportional mode for both calico node and typha separately

Reply:
Why don't we expose the two options to the config?
But, I don't want to block merging here.
I'm also fine, if it stays like this.